src/type/definition.js

```diff
@@ -566,9 +565,6 @@ export class GraphQLScalarType {
   // Parses an externally provided value to use as an input.
   parseValue(value: mixed): mixed {
     const parser = this._scalarConfig.parseValue;
-    if (isInvalid(value)) {
-      return undefined;
-    }
```

Comment:
Safe because it only used here:
https://github.com/graphql/graphql-js/blob/85971a277c4e7782e9be38d102bd6562dedf7804/src/utilities/coerceValue.js#L61-L71

So `null` and `undefined` values are never passed to `parseValue`.
Even though it's not breaking anything in `graphql-js` it's still a minor breaking change since  there is a chance that someone using `parseValue` directly.